lessons/specifics/poolboy.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: page
+title: Poolboy
+category: specifics
+order: 1
```

Comment:
Hey guys, I think @ybur-yug has a point. Perhaps this is not the kind of lesson you want in elixir school. Poolboy is not even part of the standard library. What do you think?